### ISSUE TITLE
Server crashing with: "A single server tick took 60.78 seconds (should be max 0.05)" immediately after starting

### ISSUE THREAD

**Issue by joshua-lehmann** [DarkPacks/SevTech-Ages#4528]
## Bug Description

When I try to start a new SevTech Ages server it takes a long time (10-15minutes) to start and then finaly crashes with:
A single server tick took 60.78 seconds (should be max 0.05)
Considering it to be crashed, server will forcibly shutdown.

### Did this behavior use to work in the previous version?
Not tested with a previous version, since it is a new server

## Steps to Reproduce (for bugs)
1. Created a Windows 2019 Datacenter Server on Azure (Standard D4s v3 4 vcpus, 16 GiB memory)
2. Attached static public IP Adress and opened port 25565
2. Downloaded ATLauncher and created SevTech Server
3. Starting sev tech server with following command: java -Xms4G -Xmx10G -XX:+UseG1GC -Dsun.rmi.dgc.server.gcInterval=2147483646 -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -jar forge-1.12.2-14.23.5.2854.jar %*
Also tried with the default command, same error occured

## Already tried:
Multiple restarts of the server/vm
Installed newest windows udpates
Deactivated firewall on vm

## Logs
![StartingError](https://user-images.githubusercontent.com/23728611/110203282-99b14a00-7e6d-11eb-929c-8bf8b881011c.png)

* Crash Log: [crash-2021-03-06_11.11.51-server.txt](https://github.com/DarkPacks/SevTech-Ages/files/6095144/crash-2021-03-06_11.11.51-server.txt)

## Client Information
* Modpack Version:3.2.1
* Java Version:1.8.0_281 64bit
* Launcher Used: ATLauncher
* Memory Allocated: 8GB
* Server/LAN/Single Player:Server

## World Information
* Modpack Version world created in:3.2.1
* Additional Content Installed: None

## Server Information
* Java Version:1.8.0_281 64bit
* Operating System:Windows Server 2019
* Hoster/Hosting Solution:Self hosted on Azure VM

**Comment by sam-kirby** [comment]
I’ll preface this by saying that whilst I’ve never used Azure, I hosted a SevTech server successfully on AWS for a couple of years.

Your crash report provides a snapshot of just the single instant in time when the server crashed. It can be dangerous to draw conclusions from this alone.

At the instant the max tick timer was exhausted, the server was trying to read data about a village from the world stored on disk.  You aren’t overly clear about how you’re storing the world; if you have it on a drive backed by “Azure Disk Storage”, then this kind of issue is to be expected. These kinds of disks are not directly connected to your server; they are installed in a separate server in a physically different part of the data centre and accessed over the network. This can add a huge overhead and spikes in latency will crash the server.

You are using an instance with 32GB of temporary storage which should be local to the instance. If you store the world here and back it up to Azure Disk Storage or other permanent storage then this would improve performance immensely.

**Comment by sam-kirby** [comment]
The command you give to launch the server does not include the `nogui` argument. The built in Minecraft server GUI should always be disabled as it causes significant performance bottlenecks.

Finally, and whilst this is likely unconnected to your issues, we don’t recommend using ATLauncher’s auto-generated server files, instead, please use the curated server files from Curseforge.

**Comment by joshua-lehmann** [comment]
The world is stored on a normal disk which is directly attached to the VM/Server, so not using of azure storage over internet or anything like that. Also I added the nogui to the command, however the server still had the same error always when starting.

I then tried to download and run the server instance from curseforge instead of the one form the ATLauncher. And that worked perfectly without the ticks timing outl So thanks for the tip!
